css/styles.css

```diff
@@ -0,0 +1,680 @@
+    border: 3px solid #ec5242;
+    overflow: hidden;
+  }
+
+  .emailpara {
+    width: 50%;
+    margin-left: 40%;
+    margin-top: 1%;
+    text-decoration-line: underline;
+    padding-bottom: 1%;
+    font-family: 'Lato', sans-serif;
+    font-weight: 700;
+    color: var(--black-color);
+  }
+
+  .logosection {
+    margin: auto;
+    width: 50%;
+    align-items: center;
+    text-align: center;
+  }
+
+  .prevhead {
+    margin: auto;
+    width: 50%;
+    text-align: center;
+    font-size: 1.5em;
+    font-family: 'Lato', sans-serif;
+    font-weight: 700;
+    color: var(--black-color);
+  }
+
+  .prevp {
+    margin: auto;
+    margin-top: 3em;
+    width: 50%;
+    text-align: center;
+    color: var(--black-color);
+    font-size: 1em;
+    font-family: 'Lato', sans-serif;
+    font-weight: 700;
+  }
+
+  .farmimgpartners {
+    background-color: #242525;
+  }
+
+  .sumitpara {
+    font-family: 'lato', sans-serif;
+    font-weight: 700;
+    color: #d3d3d3;
+    text-align: center;
+    font-size: large;
+  }
+
+  .sumithead {
+    text-align: center;
+    padding-top: 1em;
+    font-weight: 700;
+    color: #d3d3d3;
+    font-size: xx-large;
+    margin-top: 21px;
+  }
+
+  .speaker1,
+  .speaker2,
+  .speaker3,
+  .speaker4,
+  .speaker5,
+  .speaker6 {
+    display: flex;
+  }
+
+  .logosection h3 {
+    margin-top: 5%;
+    font-size: 1.5em;
+    font-family: 'Lato', sans-serif;
+    font-weight: 700;
+    color: var(--black-color);
+  }
+
+  .logosection hr,
+  .prevsumits hr {
+    width: 10%;
+  }
+
+  .logoimage1 img {
+    width: 36%;
+    margin-bottom: 4em;
+    margin-top: 4em;
+  }
+
+  .sumit2013-img {
+    background-color: rgba(236, 82, 66, 0.75);
+    margin: -21px 0 0 0;
+    height: 100%;
+  }
+
+  .sumit2013 {
+    width: 50%;
+    background-image: url(../img/Next-Generation-Conference-2022-group-photo-c-NFU.jpg);
+    background-repeat: no-repeat;
+    background-size: cover;
+    backdrop-filter: blur(10px);
+    height: 13em;
+  }
+
+  .sumit2014 {
+    width: 50%;
+    background-image: url(../img/PXL_20211118_092531580-scaled-e1638366981737.jpg);
+    background-repeat: no-repeat;
+    background-size: cover;
+    height: 13em;
+  }
+
+  .left-footer-text {
+    font-size: small;
+    font-family: 'Lato', sans-serif;
+    text-align: left;
+    margin: -10px 0 0 78px;
+    width: 70%;
+    color: #272a31;
+    margin-left: 32%;
+  }
+
+  .left-footer-textabout {
+    font-size: small;
+    font-family: 'Lato-Bold', sans-serif;
+    text-align: left;
+    margin: -10px 0 0 78px;
+    width: 70%;
+    color: #d3d3d3;
+    margin-left: 32%;
+  }
+
+  .footer-section {
+    height: 11em;
+  }
+
+  .footer-section-container {
+    display: flex;
+    width: 70%;
+    padding-top: 3em;
+    margin: auto;
+    margin-bottom: 5%;
+  }
+
+  .footer-sectionabout {
+    height: 9em;
+  }
+
+  .partners-bottom-content {
+    display: flex;
+    justify-content: space-between;
+    padding: 0 5px 1px 16px;
+  }
```

Comment:
- [ ] Kindly make sure to align correctly your partner's logos in order to meet the provided design. :wink: 

   - ### YOUR DESIGN
   ![image](https://user-images.githubusercontent.com/87197412/203585331-d62c5114-c0e8-487c-b960-fc0c4e505afa.png)


   - ### PROVIDED DESIGN
    ![image](https://user-images.githubusercontent.com/87197412/203585419-6abea231-2055-47d8-ab00-f506baadc43f.png)

Reply:
## issue resolved
- Thank you for noticing that part, issue is fixed.

Reply:
- Hey, great job addressing this issue as suggested by the previous reviewer, however, you made some changes, I think your alignment is not yet close enough as to be overlooked. Below is your style compared to the original design. Please, let's improve on the spaces between the partners and around the first and last partners

*your design*
![Screenshot from 2022-11-24 16-09-04](https://user-images.githubusercontent.com/59397159/203817485-19b7b1dc-a425-449d-9ffd-c7d4d6d43ea8.png)

*original design*
![Screenshot from 2022-11-24 16-08-07](https://user-images.githubusercontent.com/59397159/203817492-f6714714-d2aa-4810-91a4-4493da66b5a9.png)

- You can see the difference. it's too obvious that I can't overlook it or mark it as OPTIONAL. Please, let's improve on this to match the design as much as we can.

Reply:
- [ ] This issue is not yet resolved :x:

Reply:
## thank you for noticing that part
- I reduced the length of that part 
- then I used margin auto to fit the content